src/routes/admin/orders/frontdesk/CustomizationSelection.svelte

```diff
@@ -0,0 +1,68 @@
+<script lang="ts">
+  import { CustomizationKey, CustomizationValue } from "$lib/types";
+  import { customizationKeys, customizationValues } from "$stores/menuStore";
+  import {
+    selectedCustomizations,
+    selectCustomization,
+    initializeCustomizations
+  } from "$stores/cartStore";
+  import { onMount } from "svelte";
+
+  const getValuesByKey = (keyId: string): CustomizationValue[] => {
+    const values = $customizationValues.filter((value) => value.belongsTo === keyId);
+
+    return values.sort((a, b) => {
+      if (a.name === "Hel" || a.name === "Egen") return -1;
+      if (b.name === "Hel" || b.name === "Egen") return 1;
+      return a.name.localeCompare(b.name);
+    });
+  };
+
+  export const getSelectedCustomizationsForItem = (): CustomizationValue[] =>
+    Object.entries($selectedCustomizations)
+      .map(([, valueId]) => $customizationValues.find((v) => v.id === valueId))
+      .filter(Boolean) as CustomizationValue[];
+
+  onMount(() => {
```

Comment:
this initialization should be moved into the store, where the func is also used on reset